example/src/main/scala/example/CSRF.scala

```diff
@@ -0,0 +1,14 @@
+package example
+
+import zhttp.http.Middleware.csrf
+import zhttp.http._
+import zhttp.service.Server
+import zio.{App, ExitCode, URIO}
+
+object CSRF extends App {
+  // Will give FORBIDDEN when CSRF token in header doesn't match CSRF token set in cookies
+  val app: HttpApp[Any, Nothing] = Http.ok @@ csrf("x-token", "token")
```

Comment:
I don't think the value needs to be set by the user. It should be randomly generated internally. Only the token name is relevant for the user.

Reply:
These are the name of the header and cookie respectively.